src/main.cpp

```diff
@@ -3,7 +3,7 @@
 #define BLUE_LED 2
 
 void setup() {
-  Serial.begin(9600);
+  Serial.begin(115200);
```

Comment:
Should it be set in platformio.ini as well?

Reply:
Platformio sets the baud rate of the monitor, its not needed but good to have

Reply:
Got it... Thanks